omod/src/test/java/org/openmrs/module/fhir2/providers/TaskFhirResourceProviderTest.java

```diff
@@ -82,4 +83,22 @@ public void getTaskByWithWrongId_shouldThrowResourceNotFoundException() {
 		assertThat(resourceProvider.getTaskById(idType), nullValue());
 	}
 
+	@Test
+	public void createTask_shouldCreateNewTask() {
+		when(taskService.saveTask(task)).thenReturn(task);
+
+		MethodOutcome result = resourceProvider.createTask(task);
+		assertThat(result.getResource(), equalTo(task));
+	}
+
+	@Test
+	public void shouldUpdateTask() {
```

Comment:
We should have one testing naming convention per file. I'd probably prefer changing this one to `updateTask_shouldUpdateTask()`